packages/module/patternfly-docs/content/extensions/chatbot/examples/Analytics/Analytics.md

```diff
@@ -76,35 +80,36 @@ trackingAPI.trackSingleItem("MyEvent", { response: 'Good response' })
 
 #### Tracking providers
 
-Only providers with a matching key in the `InitProps` will be started and used.
+Only providers with a matching entry in the `InitProps.activeProviders` array will be started and used.
+
+Possible values are:
+* Umami
+* Posthog
+* Segment
+* Console
 
-```nolive
-const initProps: InitProps = {
-  segmentKey: 'TODO-key', // TODO add your key here
-  posthogKey: 'TODO-key',
-  umamiKey: 'TODO-key',
-  umamiHostUrl: 'http://localhost:3000', // TODO where is your JS provider?
-  console: true
-```
 
 ##### Modifying providers
 
 If you know upfront that you only want to use 1 of the supported providers, you can modify `getTrackingProviders()` and remove all other providers in the providers array.
 
-When using the providers you need to add additional dependencies to your package.json file:
+When using the providers you may need to add additional dependencies to your package.json file:
 
 ```nolive
 "dependencies": {
   "@segment/analytics-next": "^1.76.0",
   "posthog-js": "^1.194.4"
 ```
 
+This depends on your local setup. If you pull in
+the chatbot codebase as dependency into your project, you are already set. 
+
```

Comment:
```suggestion
Depending on your local setup, this might not be necessary. For example, if you pull the ChatBot codebase as a dependency into your project, you don't need to add it as an additional dependency in your package.json.

```